src/app/account-management/account/edit-facility/edit-facility.component.html

```diff
@@ -1,4 +1,4 @@
-<h4>Edit Facility</h4>
+<!--<h4>Edit Facility</h4>
```

Comment:
Remove commented code if not planning on adding back or referencing later. Keep if so.